src/parameterizations/lateral/MOM_tidal_forcing.F90

```diff
@@ -58,6 +67,159 @@ module MOM_tidal_forcing
 
 contains
 
+!> Finds astronomical longitudes s, h, p, and N,
+!! the mean longitude of the moon, sun, lunar perigee, and ascending node, respectively,
+!! at the specified reference time time_ref.
+!! These formulas were obtained from
+!! Kowalik and Luick, "Modern Theory and Practice of Tide Analysis and Tidal Power", 2019
+!! (their Equation I.71), which are based on Schureman, 1958.
+!! For simplicity, the time associated with time_ref should
+!! be at midnight. These formulas also only make sense if
+!! the calendar is gregorian.
+subroutine astro_longitudes_init(time_ref, longitudes_shpn)
```

Comment:
We have been thinking that is would be preferable to do our trigonometric calculations in degrees, not radians. Given that this is being newly introduced here, perhaps it would make sense to write this routine to work in degrees. Regardless, the comment describing the longitudes_shpn structure needs to explicit describe their units, preferably using the same syntax (here it would be `[rad]` or `[degrees]`) used elsewhere in the MOM6 code.

Reply:
I can make the switch from radians to degrees, but there's two concerns I have:
1. In addition to the new open boundary code, I would probably want to switch all of MOM_tidal_forcing to using degrees, including changing the tidal frequencies from rad/s to deg/s. Might change answers for any tests with tides=True? 
2. To switch to degree-based trig functions, I think we'd need to add `-fdec-math` when compiling with GNU. Not sure if flags are needed for other compilers. 

I'll be happy to make these changes, but just wanted to double check since they may not be entirely trivial.

Reply:
Alternatively I could have `eq_phase` and `nodal_fu` do calculations with astronomical longitudes in degrees, but convert the result to radians before returning. This would alleviate 1. but not 2.

Reply:
Let's not change the frequencies from rad/sec to deg/sec, since this would very likely change answers. We usually put in answer-changing code changes with a flag to allow us to revert to the previous answers, or at least we do the changes in the smallest possible code change.

What would you think, @andrew-c-ross, about having eq_phase and nodal_fu return arguments in degrees that could then be converted (or not) as needed before they are used or stored, noting that these routines only seem to be called in one place?